docs/00.md

```diff
@@ -18,13 +18,13 @@ This is a log of me going through Cats, a functional programming library for Sca
 that is currently *experimental* and under active development.
 I'll try to follow the structure of [learning Scalaz][lsz] that I wrote in 2012 (!).
 
-Cats [website][cats] says the name is a playful shortening of the word category.
-They say coordinating programmers are like herding cats
-(as in attempting to move dozens of them in a direction like cattles, while cats have their own ideas),
-and it seems to be true at least for those of us using Scala. Keenly aware of this situation, cats list
-approachability as their first motivation.
+Cats' [website][cats] says the name is a playful shortening of the word category.
```

Comment:
Here's a link on this topic http://www.chicagomanualofstyle.org/qanda/data/faq/topics/PossessivesandAttributives.html:

> Q. When indicating possession of a word that ends in s, is it correct to repeat the s after using an apostrophe? For example, which is correct: “Dickens’ novel” or “Dickens’s novel”?
>
> A. Either is correct, though we prefer the latter. Please consult 7.15–18 for a full discussion of the rules for forming the possessive of proper nouns. For a discussion of the alternative practice of simply adding an apostrophe to form the possessive of proper nouns ending in s, see paragraph 7.21.